doc/SpectatorView.Setup.md

```diff
@@ -90,7 +90,7 @@ If you are building Azure Spatial Anchors on iOS, you will need to take some add
 ### Android
 
 1. Make sure your Unity project contains the asset caches that were created in the 'Before building' steps.
-2. Import [ARCore v1.7.0](https://github.com/google-ar/arcore-unity-sdk/releases/tag/v1.7.0).
+2. Download and Import arcore-unity-sdk-1.7.0.unitypackage from [ARCore v1.7.0](https://github.com/google-ar/arcore-unity-sdk/releases/tag/v1.7.0).

```

Comment:
nit: can we add where to place this package?

Reply:
The default import location is correct.